Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewportMessages.h

```diff
@@ -205,10 +194,15 @@ namespace AzToolsFramework
             virtual float AngleStep() const = 0;
             //! Returns the current line bound width for manipulators.
             virtual float ManipulatorLineBoundWidth() const = 0;
+            //! Returns the current circle (torus) bound width for manipulators.
+            virtual float ManipulatorCircleBoundWidth() const = 0;
+
+        protected:
+            ~ViewportSettingsRequests() = default;
```

Comment:
why removing the virtual from the destructor and making it protected?

Reply:
Because no instance should ever be deleted through this pointer - buses are not treated polymorphically and so do not need a virtual destructor - by making the destructor private it means clients can't accidentally delete the object either (this is what `EBusTraits` does too)

Reply:
Got it, thanks.